WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -253,7 +238,7 @@ public async Task<CoinJoinResult> StartRoundAsync(IEnumerable<SmartCoin> smartCo
 
 			var registeredAliceClients = registeredAliceClientAndCircuits.Select(x => x.AliceClient).ToImmutableArray();
 
-			var outputTxOuts = await ProceedWithOutputRegistrationPhaseAsync(roundId, registeredAliceClients, cancellationToken).ConfigureAwait(false);
+			var (goForBlameRound, outputTxOuts) = await ProceedWithOutputRegistrationPhaseAsync(roundId, registeredAliceClients, cancellationToken).ConfigureAwait(false);
```

Comment:
I think the naming here should be according to the information that this var beholds - not according to where it will be used.

Reply:
The semantic meaning of any data is makes sense only in the context of the caller. If I rename it as `registerOutputs`   then the condition `EndRoundState.NotAllAlicesSign && goForBlameRound`  will become `EndRoundState.NotAllAlicesSign && registerOutputs`  what makes no sense for any reader.

Reply:
i changed this.